packages/core/source-map/src/SourceMap.js

```diff
@@ -0,0 +1,385 @@
+// @flow
+import {SourceMapConsumer, SourceMapGenerator} from 'source-map';
+import lineCounter from '../../utils/src/lineCounter';
```

Comment:
nit: can we rename this to `countLines` or a verb of some kind? lineCounter sounds like an object or a class.

also this should be exported at the top level of `@parcel/utils` and imported like `import {countLines} from '@parcel/utils`.

we should probably have a lint rule for requiring outside of packages with `../` since that will break on publish

Reply:
update:

> we should probably have a lint rule for requiring outside of packages with ../ since that will break on publish

did this in #3038 😃

Reply:
Can't update it as it throws eslint errors. That's why I solved it like this...`eslint(no-restricted-imports)`